### PR TITLE
[bn] Localize `content/bn/style-guide/_index.md `

### PR DESCRIPTION
**Pull request by smrafy20**
Signed-off-by: Shadman Rafy  rafyndc18@gmail.com
Localizes `content/bn/style-guide/_index.md ` to Bengali
This is an intial PR as may need some improvements!
Tracking issue: #410 #660

### CODE REVIEW

**Review by mitul3737 — APPROVED** [review]
LGTM

**Review by Imtiaz1234 — APPROVED** [review]
LGTM

File: content/bn/style-guide/_index.md [hunk 1]
```diff
@@ -7,125 +7,121 @@ menu:
...
-```
+জিনিসটি কীভাবে সমস্যার সমাধান করে তার কয়েকটি লাইন।
 
 ### Title
 
-The **title** label will always be at the top of a definition layout, and its value should be in title case. 
+*Title* লেবেল সর্বদা একটি সংজ্ঞা লেআউটের শীর্ষে থাকবে এবং এর মান শিরোনামের ক্ষেত্রে হওয়া উচিত। 
 
-```md
 ---
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:55]: Three backticks for Markdown code block are required.
(Insert ```md before this line.)

File: content/bn/style-guide/_index.md [hunk 2]
```diff
@@ -7,125 +7,121 @@ menu:
...
-title: Definition Template
-```
+title: সংজ্ঞা টেমপ্লেট
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:56]: Three backticks for Markdown code block are required.
(Insert ``` after this line.)

File: content/bn/style-guide/_index.md [hunk 3]
```diff
@@ -7,125 +7,121 @@ menu:
...
-- Not Started
+- সম্পন্ন
+- প্রতিক্রিয়া প্রশংসিত
+- শুরু না
 
-You can always open an issue against a completed definition. While a definition is in flux, its status will be changed to *Feedback Appreciated*.
+আপনি সর্বদা একটি সম্পূর্ণ সংজ্ঞার বিরুদ্ধে একটি issue খুলতে পারেন। একটি সংজ্ঞা প্রবাহিত হওয়ার সময়, এটির স্থিতি Feedback Appreciated এ পরিবর্তিত হবে।
 
-```md
 ---
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:70]: Three backticks for Markdown code block are required.
(Insert ```md before this line.)

File: content/bn/style-guide/_index.md [hunk 4]
```diff
@@ -7,125 +7,121 @@ menu:
...
-title: Definition Template
+title: সংজ্ঞা টেমপ্লেট
 status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:72]: Three backticks for Markdown code block are required.
(Insert ``` after this line.)

File: content/bn/style-guide/_index.md [hunk 5]
```diff
@@ -7,125 +7,121 @@ menu:
...
-- Technology
-- Property
-- Concept
-
-```md
+- প্রযুক্তি
+- সম্পত্তি
+- ধারণা
+- 
 ---
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:82]: Three backticks for Markdown code block are required.
(Insert ```md before this line.)

File: content/bn/style-guide/_index.md [hunk 6]
```diff
@@ -7,125 +7,121 @@ menu:
...
-title: Definition Template
+title: সংজ্ঞা টেমপ্লেট
 status: Feedback Appreciated
-category: Concept
+category: ধারণা
 ---
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:86]: Three backticks for Markdown code block are required.
(Insert ``` after this line.)

File: content/bn/style-guide/_index.md [hunk 7]
```diff
@@ -7,125 +7,121 @@ menu:
...
-4. [Omit contractions](https://en.wikipedia.org/wiki/Contraction_(grammar))
-5. [Use passive voice sparingly](https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)
-6. [Aim to phrase statements in a positive form](https://examples.yourdictionary.com/positive-sentence-examples.html) 
-7. [No exclamation marks outside of quotations](https://www.grammarly.com/blog/exclamation-mark/)
-8. Do not exaggerate
-9. Avoid repetition
-10. Be concise
+1. সহজ, সহজলভ্য ভাষা ব্যবহার করুন, প্রযুক্তিগত শব্দবাক্য এবং বাজওয়ার্ড এড়িয়ে চলুন
+2. [কথ্যভাষা এড়িয়ে চলুন](https://en.wikipedia.org/wiki/Colloquialism)
+3. [আক্ষরিক এবং কংক্রিট ভাষা ব্যবহার করুন](http://guidetogrammar.org/grammar/composition/abstract.htm)
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:16]: From #740:

I recommend to replace `http://` → `https://` .

File: content/bn/style-guide/_index.md [hunk 8]
```diff
@@ -7,125 +7,121 @@ menu:
...
 
-The Glossary aims at **explaining complex concepts in simple words** — that is a surprisingly difficult task that will likely take multiple revisions. Always keep the audience in mind when drafting your definition. Avoid using industry terms and buzzwords — you'll probably catch yourself going back to them and may need to autocorrect. 
+আপনার সংজ্ঞায় ব্যবহৃত হলে, সর্বদা *বিদ্যমান শব্দকোষের শর্তাবলীর সাথে লিঙ্ক করুন* (শুধুমাত্র প্রথম উল্লেখ হাইপারলিঙ্ক করা উচিত)।
 
-When appropriate, use **real-world examples** that help readers (especially non-technical ones) better understand *when* and *why* the concept you’re explaining is relevant. 
+*উদাহরণ*: [পরিষেবা মেশ সংজ্ঞা](https://glossary.cncf.io/service_mesh/) এর “এটি কী” বিভাগটি একবার দেখুন। এটি মাইক্রোসার্ভিস, পরিষেবা, নির্ভরযোগ্যতা এবং পর্যবেক্ষণযোগ্যতার সংজ্ঞাগুলির সাথে লিঙ্ক করে। উপরন্তু, এটি একটি মাইক্রোসার্ভিসেস পরিবেশে নেটওয়ার্ক চ্যালেঞ্জের তুলনা করে একটি বাস্তব-বিশ্বের উদাহরণ ব্যবহার করে (এমন কিছু যা অ-প্রযুক্তিগত লোকেরা সম্পর্কিত হতে পারে না)  ওয়াইফাই সমস্যার (যা কেউ ল্যাপটপ ব্যবহার করে বুঝতে পারে)সাথে । যেখানে সম্ভব, সেই সংযোগটি তৈরি করার চেষ্টা করুন।
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:110]: From https://github.com/cncf/glossary/pull/740:

I recommend to remove hardcoded site domain name:

Replace `https://glossary.cncf.io/service_mesh/` → `/service_mesh/`

File: content/bn/style-guide/_index.md [hunk 9]
```diff
@@ -7,125 +7,121 @@ menu:
...
 
-#### Start with a Google or Word doc
+#### একটি Google বা Word ডক দিয়ে শুরু করুন
 
-We recommend starting with a Google or Word doc, letting it sit for a few days, and revisiting again. This will allow you to catch phrases or expressions that could be worded in a simpler and more accessible way. Also, make sure to run a spellcheck before submitting a PR.
+আমরা একটি Google বা Word ডক দিয়ে শুরু করার পরামর্শ দিই, এটিকে কয়েক দিনের জন্য বসতে দিন এবং আবার দেখার জন্য। এটি আপনাকে বাক্যাংশ বা অভিব্যক্তিগুলি ধরতে দেয় যা একটি সহজ এবং আরও অ্যাক্সেসযোগ্য উপায়ে শব্দ করা যেতে পারে। এছাড়াও,  PR জমা দেওয়ার আগে একটি বানান পরীক্ষা চালানো নিশ্চিত করুন।
 
-To ensure no one else submits a PR while working on a term, make sure to claim an issue (or create one) and that it is assigned to you.  More to that in the [How To Contribute](https://glossary.cncf.io/contribute/) doc.
+একটি মেয়াদে কাজ করার সময় অন্য কেউ PR জমা না দেয় তা নিশ্চিত করতে, একটি সমস্যা দাবি করা (বা একটি তৈরি করুন) এবং এটি আপনাকে বরাদ্দ করা হয়েছে তা নিশ্চিত করুন। [কীভাবে অবদান রাখতে হয়](https://glossary.cncf.io/contribute/) ডক-এ আরও কিছু আছে যা দেখতে পারেন  ।
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:118]: Let's update links to the localized version if exists:

Replace `https://glossary.cncf.io/contribute/` → `/bn/contribute/`

File: content/bn/style-guide/_index.md [hunk 10]
```diff
@@ -7,125 +7,121 @@ menu:
...
+শুরু করার আগে, অনুগ্রহ করে কিছু প্রকাশিত শব্দকোষের পদ পড়ুন যাতে বিশদ এবং অসুবিধার মাত্রা এবং উদাহরণগুলি বোঝা যায়।
 
 
-## The review process: what to expect
+## পর্যালোচনা প্রক্রিয়া: কি আশা করা যায়
 
-Please note that we are currently only three maintainers doing this in their spare time. Occasionally, we'll be able to review terms quickly; on other occasions, it may take some time — we appreciate your patience. If you have any questions, please get in touch with us in the #glossary Slack channel (for where and how to find it, please refer to our [How To Contribute](https://glossary.cncf.io/contribute/) doc).
+দয়া করে মনে রাখবেন যে আমরা বর্তমানে শুধুমাত্র তিনজন রক্ষণাবেক্ষণকারী তাদের অবসর সময়ে এটি করে। মাঝে মাঝে, আমরা দ্রুত শর্তাবলী পর্যালোচনা করতে সক্ষম হব; অন্যান্য অনুষ্ঠানে, এটি কিছুটা সময় নিতে পারে — আমরা আপনার ধৈর্যের প্রশংসা করি। আপনার যদি কোনো প্রশ্ন থাকে, তাহলে অনুগ্রহ করে #glossary Slack চ্যানেলে আমাদের সাথে যোগাযোগ করুন (কোথায় এবং কীভাবে এটি খুঁজে পাবেন, অনুগ্রহ করে আমাদের [কীভাবে অবদান রাখবেন](https://glossary.cncf.io/contribute/)  ডকটি দেখুন ।
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:125]: Let's update links to the localized version if exists:

Replace `https://glossary.cncf.io/contribute/` → `/bn/contribute/`

File: content/bn/style-guide/_index.md [hunk 11]
```diff
@@ -7,125 +7,121 @@ menu:
...
 
-- Technology
-- Property
-- Concept
-
-```md
+- প্রযুক্তি
+- সম্পত্তি
+- ধারণা
+-
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:81]: Please remove this hyphen, and leave this line as an empty line.

File: content/bn/style-guide/_index.md [hunk 12]
```diff
@@ -7,125 +7,121 @@ menu:
...
-### Definition
+### সংজ্ঞা
+
+#### তিনটি উপশিরোনাম
+
+*প্রযুক্তি* এবং *ধারণা* বিভাগের সংজ্ঞায় তিনটি উপশিরোনাম রয়েছে:
 
-#### Three subheadings
+- *এটা কি*: আমরা যে বিষয়ে কথা বলছি তার একটি সংক্ষিপ্ত এবং স্পষ্ট ধারণা প্রদান করুন।
+-
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:95]: Please remove this line.

File: content/bn/style-guide/_index.md [hunk 13]
```diff
@@ -7,125 +7,121 @@ menu:
...
+
+#### তিনটি উপশিরোনাম
+
+*প্রযুক্তি* এবং *ধারণা* বিভাগের সংজ্ঞায় তিনটি উপশিরোনাম রয়েছে:
 
-#### Three subheadings
+- *এটা কি*: আমরা যে বিষয়ে কথা বলছি তার একটি সংক্ষিপ্ত এবং স্পষ্ট ধারণা প্রদান করুন।
+- 
+- *এটা যেসব সমস্যাতে ফোকাস করে*: সমস্যাটির উপর ফোকাস করুন, সমাধানের দিকে নয় (যা পরবর্তী বিভাগে আসে)। আসলে, সংজ্ঞায়িত শব্দটি উল্লেখ করা এড়িয়ে চলুন। সমস্যাটি আমাদের সেই জিনিসটির প্রয়োজন কিসের উপর তাতে আলোকপাত করে ।
+-
```
Comment (jihoon-seo) [inline content/bn/style-guide/_index.md:97]: Please remove this line.

**Review by jihoon-seo — COMMENTED** [review]
- Preview: https://deploy-preview-750--cncfglossary.netlify.app/bn/style-guide/
- en: https://glossary.cncf.io/style-guide/

Dear Bengali contributors (@smrafy20 @mitul3737 @Imtiaz1234 @Mouly22 @ikramulkayes),
please consider and reflect these suggestions!

Edit: Oh, @smrafy20 just opened another PR, and I noticed that now!

**Pull request merged**